### PR TITLE
Adds Fun

### PR DESCRIPTION
**Pull request by DaveTheHeadcrab**
SOME people seem to think I dislike fun.
SOME people seem to think my goal in life is to remove all fun.

Gives SSTs the 'Devastator' L6 saw, comes equipped with a 300-round buckshot magazine.
They spawn with a spare in their bag, and a 25-round 40mm HE magazine.

Icons:
hemag
buckshotmagh

### CODE REVIEW

File: html/changelogs/davetheheadcrab-PR-syndieLMGmadness.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a changelog editing system that should cause fewer conflicts and more accurate timestamps."
+  - rscdel: "Killed innocent kittens."
```
Comment (tigercat2000) [inline html/changelogs/davetheheadcrab-PR-syndieLMGmadness.yml]: :crying_cat_face:

**Comment by Regen1** [comment]
I approve

When is the "ANGRY MOB" L6 saw magazine, that shoots more angry SST's getting coded?

**Comment by DaveTheHeadcrab** [comment]
Damnit, forgot to save the changelog.

**Comment by NTSAM** [comment]
:ok:

File: code/modules/projectiles/ammunition/magazines.dm [hunk 2]
```diff
@@ -234,6 +234,29 @@
...
+	caliber = "shotgun"
+	ammo_type ="/obj/item/ammo_casing/shotgun/buckshot"
+	max_ammo = 300 // Bluespace!
+
+/obj/item/ammo_box/magazine/m762/buckshot/update_icon()
+	..()
+	icon_state = "[initial(icon_state)]-[round(ammo_count(),60)]"
+
+/obj/item/ammo_box/magazine/m762/he // I am not a good person.
+	name = "box magazine (40mm high explosive)"
```
Comment (Regen1) [inline code/modules/projectiles/ammunition/magazines.dm]: >Missfire in a small room and kill your entire squad rounds

Comment (DaveTheHeadcrab) [inline code/modules/projectiles/ammunition/magazines.dm]: For extra fun use it in five-round burst mode.

**Comment by Plotron** [comment]
Thank you, based Dave, the Demigod of Lesser Fun.

**Comment by TheDZD** [comment]
RIP everyone.

**Pull request merged**